src/for-developers/indexer.md

```diff
@@ -0,0 +1,13 @@
+# Fuel Indexer
+
+The Fuel Indexer is a standalone binary that can be used to index various components of the blockchain. These indexable components include blocks, transactions, and receipts and state within a Fuel network, allowing for high-performance read-only access to the blockchain for advanced dApp use-cases.
```

Comment:
- Just an FYI I copied this blurb for the Indexer book, but looks like there's a small grammatical error with an extra `and` here. 
- Should maybe be `...blocks, transactions, receipts, and state within ...` ?

Thanks @deekerno for the eagle eye 👁️

Reply:
Nice catch! I copied that blurb from [the Indexer docs](https://fuellabs.github.io/fuel-indexer/master/the-fuel-indexer.html?highlight=standalone%20binar#fuel-indexer) so I will also make a PR to fix it there!